admin_guide/custom_resource_definitions.adoc

```diff
@@ -185,28 +265,3 @@ items:
     cronSpec: '* * * * /5' <1>
     image: my-awesome-cron-image <2>
 ----
-
-[admin-guide-custom-resource-finalizers]
```

Comment:
I assume this was dropped in favor of pointing to the Kube docs?

Reply:
Yup, on line 198.